data/campaigns/Heir_To_The_Throne/scenarios/01_The_Elves_Besieged.cfg

```diff
@@ -68,7 +68,7 @@
     [side]
         type=Fighter
         id=Konrad
-        name= _"Konrad"
+        name= _"Prince Konrad"
```

Comment:
This seems weird to me - Konrad (and later Li'sar) are the player's protagonist, so there shouldn't be a formality to the user interface.

Reply:
There is precedent for that in the names of Knights and Grand Knights in S2 and S23. Also in TRoW S1. (I only played S1 and S2 of TRoW)

I added it because I thought that otherwise it wouldn't be clear whom Gryphon Tender means when she says "Your Highness".

Reply:
I'm with stevecotton on this one, adding the formality feels a bit odd in this instance, even if precedence exists elsewhere.

Reply:
OK, I'll revert it to `name= _"Konrad"`. I assume I should the same for Li'sar. Should I do that in all scenarios, or just in S16 and later when she is allied?

Reply:
Might be interesting to have her as Princess Li'sar at first, then become just Li'sar when allied. I kind of like that.

Reply:
Any other opinions before I implement this? This'll take a bit of time to implement/test because of all the different scenarios she appears in... and I'm not completely sure if her XP from S8 carries over to S16, if it does, it'll take a [modify_unit] to get this right.

Reply:
Having her change from Princess Li'sar to Li'sar sounds good to me.

The name does carry over from S8 to S16, 7049367bf073a6e35530b97eba1cf79af86dc4a7 included a bugfix for a typo in her name in the NEED_LISAR macro, which only had an effect if there wasn't already a copy of her from S8.